### ISSUE TITLE
Game crashing near barakoa villages

### ISSUE THREAD

**Issue by Jwilson106** [JoelZBub/organic-free-range-pack#75]
Describe the Bug: 
While trying to find an area on my map that isn't a desert I came across a Barakoa village and when I get near enough that the barakoa run up to attack me my game freezes and crashes to the launcher showing exit code 0.

How Can We Reproduce the Bug:
1. Find a Barakoa village near the desert (Not sure is biome makes a difference but that's all I can find so far).
2. Walk/teleport close enough for the barakoa to begin attacking.

Expected behavior:
I should be able to walk through the village and fight them without the game crashing.

If Applicable, Add Screenshots or Video to Explain the Bug:
No crash reports show up in the game folder

Paste your crash report or log file to https://hatebin.com/ and paste the link here.
Please complete the following information:

    Have you added or removed any mods, and if so, which?: No mods removed.
    Modpack Version Number : 1.1.1
    Is OptiFine installed? : No.
    Does this issue affect Server, Client, or Both? : Client only so far not sure about server

**Comment by JoelZBub** [comment]
Need some kind of crash report or log to diagnose this one.

**Comment by Jwilson106** [comment]
While trying to get video of what happens I did get this crash log but I'm not sure if it is from me being near the village or if it is another issue all together.
https://hatebin.com/mldienldeq

**Comment by Jwilson106** [comment]
This is as close as I could get to a video of what's happening. I was able to get what's left of my items from my body and when I went around the house and a barakoa saw me that's where the gif picks up.

![ezgif com-gif-maker(1)](https://user-images.githubusercontent.com/63497105/93006699-8653a080-f514-11ea-9e9f-018bbeaba07e.gif)

**Comment by JoelZBub** [comment]
The crash is related to Xaero's map. 
Please disable Xaero's before starting your game and see if you can get a crash.
Also, hatebin.com has started limiting character counts, so not all of your crash report was posted. Please repost it to https://gist.github.com/ and share.

**Comment by Jwilson106** [comment]
I tried disabling Xaero's map in the main menu but it wouldn't disable in the mods menu so I checked in the pack folder and didn't see a config file for it so I'm not sure how to proceed on that front.
I also tried opening a test world with creative enabled and between spawning in mobs and finding a village in the world and running in in survival mode I wasn't able to recreate the issue with the game freezing when they attack me so it may be an issue with that specific world so I'm fine with scrapping it and playing in a new sp world so we can close this as a random one off glitch.
